website/docs/r/sentinel_watchlist_item.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_watchlist_item"
+description: |-
+  Manages a Sentinel Watchlist Item.
+---
+
+# azurerm_sentinel_watchlist_item
+
+Manages a Sentinel Watchlist Item.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-workspace"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_solution" "example" {
+  solution_name         = "SecurityInsights"
+  location              = azurerm_resource_group.example.location
+  resource_group_name   = azurerm_resource_group.example.name
+  workspace_resource_id = azurerm_log_analytics_workspace.example.id
+  workspace_name        = azurerm_log_analytics_workspace.example.name
+  plan {
+    publisher = "Microsoft"
+    product   = "OMSGallery/SecurityInsights"
+  }
+}
+
+resource "azurerm_sentinel_watchlist" "example" {
+  name                       = "example-watchlist"
+  log_analytics_workspace_id = azurerm_log_analytics_solution.example.workspace_resource_id
+  display_name               = "example-wl"
+}
+
+resource "azurerm_sentinel_watchlist_item" "example" {
+  name         = "0aac6fa5-223e-49cf-9bfd-3554dc9d2b76"
+  watchlist_id = azurerm_sentinel_watchlist.example.id
+  fields = {
```

Comment:
fields is different then many other resources? would this not be more consistent across the provider as
```suggestion
  properties = {
```